Add docs for EVM avax.getAtomicTxStatus. (#179)
ava-labs/avalanche-docs: build/avalanchego-apis/contract-chain-c-chain-api.md

}
```

### avax.getAtomicTxStatus

Get the status of an atomic transaction sent to the network.

#### **Signature**

```cpp
avax.getAtomicTxStatus({txID: string}) -> {
  status: string,
  blockHeight: string // returned when status is Accepted
}
```

`status` is one of:

* `Accepted`: The transaction is \(or will be\) accepted by every node. Check the `blockHeight` property
* `Processing`: The transaction is being voted on by this node

> * `Dropped`: The transaction will never be accepted by any node in the network

Reviewer: ```suggestion
* `Dropped`: The transaction was dropped by this node because it thought the transaction invalid
```
